src/js/components/modals/ModalHeading.d.ts

```diff
@@ -0,0 +1,11 @@
+import { ReactElement } from "react";
+
+type HeadingLevel = 1 | 2 | 3 | 4 | 5 | 6;
```

Comment:
this translates to string, correct? (in js component)

Reply:
According to propTypes those should be numbers. Then they translate to strings